mails/themes/modern/core/account.html.twig

```diff
@@ -248,7 +248,7 @@
                             <tbody>
                               <tr>
                                 <td align="center" style="font-size:0px;padding:10px 25px;word-break:break-word;">
-                                  <div style="font-family:Open sans, arial, sans-serif;font-size:14px;line-height:25px;text-align:center;color:#363A41;">{{ 'You can now place orders on our shop:'|trans({}, 'Emails.Body', locale)|raw }} <a href="{shop_url}" target="_blank" style="text-decoration:none;color:#25B9D7; display:block">{shop_name}</a></div>
+                                  <div style="font-family:Open sans, arial, sans-serif;font-size:14px;line-height:25px;text-align:center;color:#363A41;">{{ 'You can now place orders on our store:'|trans({}, 'Emails.Body', locale)|raw }} <a href="{shop_url}" target="_blank" style="text-decoration:none;color:#25B9D7; display:block">{shop_name}</a></div>
```

Comment:
Hi @lartist,

sorry but this is a special case (again 😅) the templates for the modern theme are actually generated, they are based on MJML templates, a markup language for email templates which is then compiled. This is not blocking for the PR don't worry, the wordings can be updated manually as a temporary solution.

However, if we don't want these wordings to be overridden the initial sources must be updated as well, and exported in the core in a second PR to make sure they are fully synced.

Reply:
You can find a bit of details in this mail theme README https://github.com/PrestaShop/PrestaShop/blob/develop/mails/themes/modern/README.md

This is the tool that compiles MJML templates into twig ones https://github.com/PrestaShop/mjml-theme-converter You will have to update the content of the source theme which is in the same repo as the tool https://github.com/PrestaShop/mjml-theme-converter/tree/master/mails/themes/modern_mjml A PR on the repo will be needed and in parallel, you'll have to run the command (detailed in the README) to update the modern theme in the core repository (and make another PR)

Reply:
I hope the explanation are clear enough, don't hesitate to ask for more details if needed

Reply:
ok